Resources/Core/Build/Configuration/Acceptance/Support/AcceptanceTester.php

```diff
@@ -53,5 +53,9 @@ public function useExistingSession()
 
         // reload the page to have a logged in backend
         $I->amOnPage('/typo3/index.php');
+        
+        // Ensure main content frame is fully loaded, otherwise there are load-race-conditions
+        $I->switchToIFrame('list_frame');
+        $I->waitForText('Web Content Management System');
```

Comment:
we should also add $I->switchToIFrame(); here, so the default is to be on the main iframe after default login.